pkg/client/unversioned/restclient.go

```diff
@@ -88,6 +89,9 @@ func (c *RESTClient) Verb(verb string) *Request {
 	if c.Throttle != nil {
 		c.Throttle.Accept()
 	}
+	if c.Client == nil {
+		return NewRequest(nil, verb, c.baseURL, c.apiVersion, c.Codec)
+	}
```

Comment:
More info on how this usually bites people: https://golang.org/doc/faq#nil_error